contrib/examples/actions/cloudslang-basic.yaml

```diff
@@ -0,0 +1,13 @@
+---
+description: Greet StackStorm from cloudslang ;) !
+enabled: true
+entry_point: ""
+name: cloudslang-basic
+pack: examples
+parameters:
+  flow_path:
```

Comment:
examples pack gets installed into `/opt/stackstorm/pack/examples` so you should be able to use that path.